godtools/Views/TractElements/TractButton.swift

```diff
@@ -78,6 +82,24 @@ class TractButton: BaseTractElement {
         self.addSubview(self.button)
     }
     
+    override func buildFrame() -> CGRect {
+        return CGRect(x: self.xPosition,
+                      y: self.yPosition,
+                      width: self.width,
+                      height: self.height)
+    }
+    
+    override func textStyle() -> TractTextContentProperties {
+        let textStyle = super.textStyle()
+        textStyle.font = .gtRegular(size: 18.0)
```

Comment:
It should be supported, I believe it was talked on a meeting (I have to wait for that, so I can make some changes on some properties classes)